apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.mustache

```diff
@@ -67,7 +67,34 @@ $(function() {
         </div>
         <ul class="card-body list-group">
             {{#student_submission}}
-            	<li class="{{block_formatting}}" style="margin-left: {{indent}}px">{{{inner_html}}}</li>
+                {{#distractor_feedback}}
+                <li class="pl-order-blocks-pairing-indicator list-group-item-warning">
+                    <ul class="p-0">
+                        <li class="list-unstyled">
```

Comment:
Nit: check out the Bootstrap docs. This class should be on the `<ul>` element, and it will automatically remove the padding (so no need for `p-0`).

https://getbootstrap.com/docs/4.6/content/typography/#unstyled

Reply:
done, thanks